config.yaml

```diff
@@ -5,3 +5,12 @@ options:
   cluster-name:
     description: "Optional - Name of the MySQL InnoDB cluster"
     type: "string"
+# Config options for the legacy 'mysql relation'
+  user:
+    default: "mysql"
+    description: "The database username"
```

Comment:
I think its useful to describe that this is only used for legacy and it will be deprecated, and the same on L15

Reply:
Addressed in 16a0a25a5b7c82698b3fa16c84266763cb9a4193